aep/general/0157/aep.md.j2

```diff
@@ -37,15 +40,41 @@ Field masks **should not** be specified in the
   [here](https://github.com/protocolbuffers/protobuf/issues/8547#issuecomment-2005180068).
   Consider using the view enumeration pattern described below instead instead.
 
-**Note:** Changing the default value of the field mask parameter is a
-[breaking change](./backwards-compatibility#semantic-changes).
+{% tab oas %}
+
+**Note:** OAS example not yet written.
+
+{% endtabs %}
 
 ### View enumeration
 
 Alternatively, an API **may** support partial responses with view enums. View
 enums are useful for situations where an API only wants to expose a small
 number of permutations to the user:
 
+- The enum **should** be specified as a `view` field on the request message.
+- The enum **should** be named something ending in `-View`
```

Comment:
This is probably proto-specific guidance - enums aren't named in OpenAPI - they're lists of strings.

Reply:
Good call, moved.